src/FSharpSource.targets

```diff
@@ -185,6 +185,7 @@
     <DefineConstants>$(DefineConstants);NETSTANDARD1_6</DefineConstants>
     <DefineConstants>$(DefineConstants);NETCOREAPP1_0</DefineConstants>
     <OtherFlags>$(OtherFlags) --simpleresolution</OtherFlags>
+    <TargetProfile>netcore</TargetProfile>
```

Comment:
As don said, the `TargetFramework` (and `TargetFrameworks` ) depends on on .net core sdk msbuild targets.
Atm i pass `--targetprofile:netcore` flag to fsc (in sdk `preview2` too) if target framework is `netstandard` or `netcoreapp` ( ref https://github.com/dotnet/netcorecli-fsc/blob/b5dd998bc6d444812c62cbac96eae48078eb77f7/src/dotnet-compile-fsc/Program.cs#L179-L182 )
Maybe that is another thing the FcsTask can infer by itself becased on TargetFramework